scripts/ubuntu-bionic/10-v4l2loopback.sh

```diff
@@ -15,7 +15,14 @@ fi
 
 apt-get install v4l2loopback-dkms -y
 
-if [[ "$CLOUD" == "google" ]]; then
+if [[ "$TASKCLUSTER_INSTANCE" == "firefoxci" ]]; then
```

Comment:
The logic flow here should be "if v4l2loopback should be built from source, then build it from source" rather than "if we are running in firefox-ci, then build v4l2loopback from source". In other words, using a flag such as BUILD_VL412LOOPBACK_FROM_SOURCE.

It isn't clear to me from the PR under what conditions this is needed, only from the code that it seems to happen under gcp/firefox-ci. It would be useful to cite the explicit conditions under which it is required to build from source, ideally with a link to further information / bug etc.

The setting of BUILD_VL412LOOPBACK_FROM_SOURCE can then be applied to files in the vars directory as appropriate. In those files, a comment can be added to say what specifically about the context (environment/cloud/etc) triggers the requirement.

An explicit flag such as BUILD_VL412LOOPBACK_FROM_SOURCE that is enabled or disabled allows the concerns between what the feature does, and when it is needed, separate, which should help when it needs to be enabled/disabled in some other environment, or environments are no longer needed/used etc.